projects/android/.gitignore

```diff
@@ -0,0 +1,9 @@
+*.iml
+.gradle
+/local.properties
+/.idea
+.DS_Store
```

Comment:
`.DS_Store` should probably be in the top-level `.gitignore`.

Reply:
This file was generated by Gradle, but I agree .DS_Store should be in the root .gitignore. I will move it.